src/client/components/UsersListComponent/UsersList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './UsersListStyle.css';
+
+export default function Userslist(props) {
```

Comment:
This component does too much. It should only be the UserList. You have also added the h1, the input etc. Only add the ul and li as part of this component. Have you talked to Deepthi about how to get users? https://github.com/HackYourFuture-CPH/chattie/issues/29